src/actions/guides/http_and_routing/sessions_and_cookies.cr

```diff
@@ -82,7 +82,49 @@ class Guides::HttpAndRouting::SessionsAndCookies < GuideAction
 
     # Set the cookie to HTTP only
     c_is_for_cookie.http_only(true)
+
+    # Set the cookie SameSite to Lax. 
+    # You can also use `:strict` here.
+    c_is_for_cookie.samesite(:lax)
+    ```
+
+    ### Setting cookie defaults
+
+    The `Lucky::CookieJar` configuration is located in your app's `config/cookies.cr` file.
+    You can set defaults for your cookies when they're written.
+
+    ```crystal
+    # config/cookies.cr
+    Lucky::CookieJar.configure do |settings|
+      settings.on_set = ->(cookie : HTTP::Cookie) {
+        cookie
+          .secure(true)
+          .http_only(true)
+          .samesite(:strict)
+          .expires(1.year.from_now)
+          .domain("mydomain.com")
+      }
+    end
+    ```
+
+    ## Disabling Cookies
+
+    There may be an action where you need disable writing cookies. For this, you
+    can use the `disable_cookies` macro in the action you want to skip writing cookies.
+
+    ```crystal
+    class Api::Posts::Index < ApiAction
+      disable_cookies
+
+      get "/posts" do
+        json([] of Post)
+      end
+    end
     ```
+
+    This will disable writing all cookies, session, and flash messages for this action.
+    If you need to do this for all of your API, you can add the `disable_cookies` macro to
+    your `ApiAction`. Then all actions that inherit from ApiAction will skip writing cookies.
```

Comment:
```suggestion
    This will disable writing all cookies, session, and flash messages for this action.
    If you need to do this for all of your actins, you can add the `disable_cookies` macro to
    the action you inherit from (for example the `ApiAction`).
```